### ISSUE TITLE
`TrimMultilineRawString` false-positive on annotation parameters

### ISSUE THREAD

**Issue by BraisGabin** [detekt/detekt#5476]
## Observed Behavior
```kotlin
    @DisplayName(
        """
            Given something
            When something
            Then something
        """,
    )
```

`TrimMultilineRawString` reports raw strings inside annotations. The problem is that the compiler doesn't allow to add `.trimIndent()` on that scope so we should disable this rule for this case.

## Your Environment
* Version of detekt used: 1.22.0-RC2
